usecases/objects/add.go

```diff
@@ -33,10 +33,10 @@ type schemaManager interface {
 	) (*models.Class, error)
 	// ReadOnlyClass return class model.
 	ReadOnlyClass(name string) *models.Class
+	// AddClassProperty it is upsert. it adds properties to a class and updates
```

Comment:
non-blocking(typo/suggestion): `AddClassProperty it is upsert` -> `AddClassProperty is an upsert operation`.